presenter/src/components/language/LanguageSelector.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <v-select
+    v-model="$i18n.locale"
+    density="compact"
+    name="language"
+    :items="languages"
+    item-value="locale"
+    class="language-select"
+    variant="solo"
+    base-color="#3D4753"
+    color="#3D4753"
+    bg-color="transparent"
+    hide-details="auto"
+    flat
+  >
+    <template #selection="{ item }">
+      <span>{{ item.raw.locale.toUpperCase() }}</span>
+    </template>
+    <template #item="{ props }">
+      <v-list-item v-bind="props" class="language-item" active-class="language-item-active">
+        <template #title="{ title }">
+          <div class="language-item-title">{{ title }}</div>
+        </template>
+      </v-list-item>
+    </template>
+  </v-select>
+</template>
+
+<script lang="ts" setup>
+import { ref } from 'vue'
+
+// TODO better place? maybe locales folder?
+const languages = ref([
+  { locale: 'de', title: 'Deutsch' },
+  { locale: 'en', title: 'Englisch' },
```

Comment:
```suggestion
  { locale: 'en', title: 'English' },
```

Shouldn't it be `English` which is English while `Englisch` is German.